sympy/polys/polytools.py

```diff
@@ -1818,6 +1818,25 @@ def total_degree(f):
         else:  # pragma: no cover
             raise OperationNotSupported(f, 'total_degree')
 
+    def power_list(f):
+        """
+        Returns a list of powers of variables in ``f``.
+
+        Examples
+        ========
+
+        >>> from sympy import Poly
+        >>> from sympy.abc import x, y
+
+        >>> Poly(x**2 + y*x + 1, x, y).power_list()
+        (2, 1, 0)
+
+        """
+        if hasattr(f.rep, 'power_list'):
+            return f.rep.power_list()
+        else:  # pragma: no cover
```

Comment:
`else` not needed.

Reply:
Was based on old code structured followed but i removed else in next commits to improve code quality in next commits. 
May be you can open a PR to remove unnecessary else written in polytools file to improve code quality.